packages/inter-protocol/src/price/priceAggregatorChainlink.js

```diff
@@ -320,9 +316,33 @@ export const start = async (zcf, privateArgs, baggage) => {
     getPriceAuthority() {
       return priceAuthority;
     },
-    getSubscriber: () => quoteSubscriber,
+    /**
+     * @deprecated use getPublicTopics
+     * @returns {Subscriber<PriceDescription>}
+     */
+    getSubscriber: () => {
```

Comment:
`getSubscriber()` and `getPublicTopics()` are new. Worth adding some tests?  Are there tests using old approaches that can be updated?

Reply:
`getSubscriber` isn't new. The diff might look like it is because of the new jsdoc.

`getPublicTopics` is new and it's covered in integration tests with smart wallet, the thing that uses it.

But yeah if I'm saying to deprecate `getSubscribers` then the new one should have a test per contract. Will add.

Reply:
fortunately it was already covered by virtue of using the `topicPath` helper in its [storage keys tests](https://github.com/Agoric/agoric-sdk/blob/9883ed6937ac5b0fd73fabb5073c85ce93afd346/packages/inter-protocol/test/test-priceAggregatorChainlink.js#L766-L776).